rpms/SPECS/wazuh-manager.spec

```diff
@@ -843,6 +843,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Mon Sep 01 2023 support <info@wazuh.com> - 4.6.0
```

Comment:
Sep 01 is Friday. Fix it please.

Reply:
Done [2140283](https://github.com/wazuh/wazuh-packages/pull/2068/commits/2140283ee1f7fc02016a09d2b4e08f26a2758c07)